simplq/src/components/common/QueueStats/QueueStats.jsx

```diff
@@ -1,5 +1,9 @@
-import React, { useMemo } from 'react';
+import React, { useEffect, useCallback, useMemo } from 'react';
 import moment from 'moment';
+import { useGetQueueStatus } from 'store/asyncActions';
+import { selectQueueStatus } from 'store/queueStatus';
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
Maybe we should rename selectQueueStatus to something like selectQueueDetails. In the pauseQueue PR, we have a selector with the same name and it makes more sense there.

Reply:
Also useGetQueueStatus to useGetQueueDetails since we have useSetQueueStatus coming in that PR too.

Reply:
Yeah, right. What to do think about "QueueInfo" here and for corresponding API and slices and component for this?

This API gives "info" about the queue, the components shows queue "information" etc..

Stats -> Info

Reply:
Yeah, sounds good

Reply:
I've handled it as a separate PR here: https://github.com/SimplQ/simplQ-frontend/pull/593